components/dashboard_message.py

```diff
@@ -0,0 +1,17 @@
+from components.message import Message
+from canvas import Canvas
+from data import Data
+
+
+class DAShboardMessage(Message):
+    """Displays messages sent from the DAShboard onto the camera overlay.
+
+    Messages are only sent when the overlay is connected to the DAS.
```

Comment:
"Messages can only be received when the overlay is connected to the DAS" might be more accurate.